src/hu_lib.c

```diff
@@ -519,7 +519,10 @@ void HUlib_erase_widget (const hu_widget_t *const w)
   const hu_font_t *const f = *m->font;
 
   const int height = m->numlines * f->line_height;
-  const int y = vert_align_widget(w, m, f, w->h_align, w->v_align);
+  int y = vert_align_widget(w, m, f, w->h_align, w->v_align);
```

Comment:
Hm, I think you need to call `vert_align_widget()` `m->numlines` times. Else the next widget after the multiline widget will not get properly cleaned.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1628/commits/5f7e4683dbbabecc099b3f6ed8476d8e65ca2538